client/lobbies/race-picker.jsx

```diff
@@ -87,27 +88,31 @@ export const StyledRaceIcon = styled(RaceIcon)`
 export default class RacePicker extends React.Component {
   static propTypes = {
     race: PropTypes.oneOf(['r', 'p', 't', 'z']).isRequired,
+    disableRace: PropTypes.oneOf(['r', 'p', 't', 'z']),
```

Comment:
If you made this take a list instead of just 1 race, I think you could have `SelectedRace` forward completely to this component as well, might be a nice way to clean up the various implementations of this

(might be better to name it `hiddenRaces` or something in that case, since everything in `SelectedRace` is non-interactable)